src/bn.iced

```diff
@@ -26,9 +26,9 @@ toMPI = (bn) ->
   ba = bn.toByteArray()
   # The top byte isn't a full byte, so figure out how many bits it takes
   size = (ba.length - 1) * 8 + nbits(ba[0])
-  hdr = new Buffer 2
+  hdr = Buffer.alloc(2)
```

Comment:
note that the danger of this PR is to mix up `Buffer.from` and `Buffer.alloc`, but otherwise, it's a straight `sed` operation.